index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>A Meaningful Page Title</title>
+    <link href="style.css" rel="stylesheet" />
+  </head>
+
+  <body>
+    <div id="main">
```

Comment:
- [ ] Change `div`  with `main` tag.
 Using semantic HTML is important for accessibility :muscle: